src/mscorlib/shared/System/Runtime/CompilerServices/IntrinsicAttribute.cs

```diff
@@ -6,7 +6,7 @@ namespace System.Runtime.CompilerServices
 {
     // Calls to methods or references to fields marked with this attribute may be replaced at
     // some call sites with jit intrinsic expansions.
-    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Constructor | AttributeTargets.Field, Inherited = false)]
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Constructor | AttributeTargets.Field | AttributeTargets.Property, Inherited = false)]
```

Comment:
This does not make sense on property. It has to be on the method implementing the property, like:

```
public static bool IsSupported
{
    [Intrinsic]
    get { return false; }
}
```

Reply:
Will fix.